narwhals/_pandas_like/expr.py

```diff
@@ -456,10 +457,12 @@ def func(df: PandasLikeDataFrame) -> list[PandasLikeSeries]:
                         "na_option": "keep",
                         "pct": False,
                     }
-                else:  # Cumulative operation
+                elif function_name.startswith("cum_"):  # Cumulative operation
                     if self._call_kwargs["reverse"]:
                         raise NotImplementedError(unsupported_reverse_msg)
                     kwargs = {"skipna": True}
+                else:
+                    kwargs = {}
```

Comment:
How explicit do we want to be here? Should we have an `elif` case for `function_name == "diff"` or is that ok to have a final else case?